packages/google_atlas/pubspec.yaml

```diff
@@ -16,7 +16,8 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  atlas: ^0.7.0
+  atlas:
+    path: ../atlas
```

Comment:
is this right?

Reply:
> is this right?

I think so because the google-atlas project are in the same project so this way it will always use the last version of Atlas, right?
What do you think about it?

Reply:
They are in the same repository (monorepo) but the projects are different. What happen if I want to use a different version of atlas?

Reply:
> They are in the same repository (monorepo) but the projects are different. What happen if I want to use a different version of atlas?

Yeah, I think you are right, so I just remembered that I need also to change the google-atlas version, I did already, please review again.

Reply:
> > They are in the same repository (monorepo) but the projects are different. What happen if I want to use a different version of atlas?
> 
> Yeah, I think you are right, so I just remembered that I need also to change the google-atlas version, I did already, please review again.

After Changed to a specific version I realized that it won't work because it depends on the Atlas new version deploy.
So regarding your last question, I thin we can reference the Atlas path in Google-Atlas because we are changing both versions at the same time and Google-atlas implementation depends on Atlas changes.

I would like to see what @dakrawczyk  and @tchabot22 think about that.

Reply:
@edsondiasalves the `atlas` dependency should not be resolved with a path, but with the version.

You will need to make the change in atlas, publish atlas, and then consume this new version of atlas in google_atlas.